README.md

```diff
@@ -340,6 +341,30 @@ be run multiple times and not just in the context of the concrete test classes,
 
 Important: One should not override test methods inherited from a base class and then call `super()` as this will run the tests from the base class twice.
 
+Best practice: Do not inherit from other classes that have been decorated with the ``suite`` decorator. Instead use abstract test base classes.
+
+### Overriding Tests
+Sometimes you might want to override tests inherited from a given base class. You can do this by redeclaring the same test method in your sub class, e.g.
+``` TypeScript
+export abstract class AbstractTestBase {
+
+  @test 'test that will be overridden by sub classes'() {
+
+    chai.assert.fail('sub classes must override this');
+  }
+}
+
+export class ConcreteTest extends AbstractTestBase {
+
+  @test.skip 'test that will be overridden by sub classes'() {
+  }
+}
+```
+
+You may now either implement the test or simply just skip it, and so on.
+
+Given that ``skip`` actually marks the test as pending, this might not be what you want for your test reports. In that case, just override the test with an empty body.
```

Comment:
I think if you override a base test but omit the `@test` decorator, the test will be completely hidden. It may be worth mentioning here.

Reply:
Actually, it does not. The part of the code that collects the test methods from the different prototypes checks whether test test symbol is set on the method. If it is not then it will just skip the method.